Unicode/utype.c

```diff
@@ -41001,7 +41001,7 @@ const uint32 ____utype[] = { 0,
   0x180c2010, 0x00080000, 0x00004000, 0x00004000, 0x00004000, 0x00004000, 0x00000000, 0x00000000,
   0x10000000, 0x00000000, 0x10200100, 0x00000000, 0x00000000, 0x00020000, 0x00000000, 0x10000000,
   0x00004000, 0x00004000, 0x10000400, 0x10000400, 0x10010000, 0x10200101, 0x00000000, 0x00000000,
-  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x10000000, 0x10000000, 0x10000000, 0x00080000,
+  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x30000000, 0x30000000, 0x30000000, 0x00080000,
```

Comment:
I heard complaints of how some changes made the fontforge-fork slower in some respects.
As there are approx only 600 values of interest, that means there are approx 65000 values that aren't of interest. Dedicating 1bit as a flag (8kbytes) allows you to avoid doing a bsearch on the 65000 other values that are not ligatures, vulgars, or fractions as you already know they are, or are not a code of interest.